sdk/search/azure-search-documents/azure/search/documents/_internal/_search_client.py

```diff
@@ -266,6 +367,36 @@ def autocomplete(self, search_text, suggester_name, **kwargs):
         :param str search_text: The search text on which to base autocomplete results.
         :param str suggester_name: The name of the suggester as specified in the suggesters
         collection that's part of the index definition.
+        :keyword autocomplete_mode: Specifies the mode for Autocomplete. The default is 'oneTerm'. Use
```

Comment:
In .NET we just have this as `mode`. @tg-msft, should it match in this case or is this idiomatically acceptable?

/cc @johanste

Reply:
I am ok to use mode, just want to confirm because autocomplete_mode is from swagger.